portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -22,6 +22,7 @@
   public abstract long id();
   public abstract String name();
   public abstract String email();
+  public abstract String imageURL();
```

Comment:
See my comments from your other PR, this needs to be named imageUrl(). Also, should it be optional?

Reply:
made optional